Customer Support Procedures/Neo4j-v5.0-migration.md

```diff
@@ -0,0 +1,47 @@
+<!-- omit in toc -->
+# Neo4j migration to v5 guidelines
+
+If you are planning to migrate your Neo4j database from an older version to v5,
+this document will provide you extra details of what you need to take care of in your Linkurious Enterprise project.
+
+<!-- omit in toc -->
+# Table of contents
+- [Pre-requisites and limitations](#pre-requisites-and-limitations)
+- [Issues with cypher queries](#issues-with-cypher-queries)
+- [Migrating databases](#migrating-databases)
+
+# Pre-requisites and limitations
+
+- Linkurious Enterprise supports Neo4j v5 starting from the version v4.0.1, if you are working with an older version of Linkurious Enterprise
+update it before updating Neo4j (Linkurious Enterprise [update procedure](https://doc.linkurio.us/admin-manual/latest/update-procedure/))
```

Comment:
The https://doc.linkurio.us/admin-manual/latest/update-procedure/ will point to the doc of 3.1. If the squad is not planning any upgrade to the doc install (I'm not aware of any yet), I think it's fine. Otherwise be sure to link the future v4.0 documentation

Reply:
I think it's fine. We can change it later in any case.